README.md

```diff
@@ -0,0 +1,304 @@
+programming quiz. 
+
+#### Features
+
+ * Define fairly complicated programming problems and have users
+ solve the problem. 
+ * Immediate verification of code solution. 
+ * Supports pretty much arbitrary coding questions in Python, C, C++ and 
+ simple Bash and uses "test cases" to test the implementations of the students.
+ * Supports simple multiple choice questions and File uploads.
+ * Since it runs on your Python, you could technically test any Python based library.
+ * Scales to over 500+ simultaneous users.
+ * Distributed under the BSD license.
+
+Quick Start
+===========
+
+#### Pre-Requisite
+
+1. Ensure [pip](https://pip.pypa.io/en/latest/installing.html) is installed
+
+#### Installation
+
+1. Install the vimarsh app
+    ```pip install vimarsh```
+
+1. In the terminal run ```vimarsh create_demo [-p PATH] project_name```
+	- project_name is the name of the django project 
+    - PATH is an optional argument to specify where the django project will be installed
+	- If PATH is not provided, the project is created in the current directory
+
+1. The script does the following;
+	1. Creates a new django project called <projectname>
+	1. Creates a new demo database
+	1. Creates two users, test moderator and test examinee
+	1. Loads demo questions
+	1. Loads demo quiz
+
+1. Run ```vimarsh run_demo```
+
+1. In a new terminal run ```sudo vimarsh run_code_server```
+
+1. Open your browser and run http://localhost:8000/exam
+
+1. Login as a teacher to edit the quiz or as a student to take the quiz
+	Credentials:
+	- Student - Username: student | Password: student
+	- Teacher - Username: teacher | Password: teacher
+
+Production Deployment
+======================
+
+#### Pre-Requisite
+
+ 1. Install dependencies using ```pip install -r requirements.txt```
+ 1. Install MySql Server
+ 1. Install Python MySql support
+ 1. Install Apache Server for deployment
+ 
+#### Configure MySql server
+
+ 1. Create a database named ``online_test`` by following the steps below
+    
+    1. $> mysql -u root -p
+    
+    1. mysql> create database online_test
+ 
+ 1. Add a user named ```online_test_user``` and give access to it on the database ```online_test``` by following the steps below
+ 
+    1. mysql> grant usage on online_test.* to online_test_user@localhost identified by 'mysecretpassword';
+    
+    1. mysql> grant all privileges on online_test.* to online_test_user@localhost;
+ 
+ 1. Create a file named `local.py` in folder `testapp` and insert `DATABASE_PASSWORD = 'mysecretpassword'`
+
+To deploy this app follow the steps below:
+
+ 1. Clone this repository and cd to the cloned repo.
+  		```git clone  https://github.com/FOSSEE/online_test.git```
+
+ 1. Run:
+ 
+        django syncdb
+ 
+    1. Run:
+    
+            django migrate exam
+    
+ 1.  Add questions by editing the "docs/sample_questions.py" or any other file in the same format and then run the following:
+
+        django load_exam docs/sample_questions.py
```

Comment:
same here
python manage.py load_exam docs/sample_questions.py
or simply say
python manage.py load_exam <path to sample_questions.py>